src/OrbitGl/PrimitiveAssembler.h

```diff
@@ -98,6 +98,9 @@ class PrimitiveAssembler {
 
   void AddCircle(const Vec2& position, float radius, float z, Color color);
 
+  void AddVerticalArrow(float x, float y_start, float y_end, float z, uint32_t width,
```

Comment:
I would be consistent with `AddVerticalLine` here, so using `pos` and `size` (or `lenght` if you just want to include the vertical lenght). But using `pos` and `size` could help you to not need to include `width` and saving a parameter.

Reply:
I get this but the ending position could be higher or lower than the starting position. I would only get xs and ys for the thread slices and I would like to use them as is, instead of having to do some calculations before calling the function. 

I think I'll just overload the function with a signature similar to vertical line instead of changing this one.

Reply:
I think it's better to do the calculations in your method and use just a standardized method here. I don't have a strong opinion between this and the VerticalLine but yes about being consistent and using the same pattern in both places.